src/hooks/useSystemIntakeContacts.tsx

```diff
@@ -101,66 +129,90 @@ function useSystemIntakeContacts(
     deleteSystemIntakeContact
   ] = useMutation<DeleteSystemIntakeContactInput>(DeleteSystemIntakeContact);
 
-  /** Create system intake contact in database */
+  /**
+   * Create system intake contact in database
+   * */
   const createContact = async (contact: SystemIntakeContactProps) => {
     const { euaUserId, component, role } = contact;
-    return createSystemIntakeContact({
-      variables: {
-        input: {
-          euaUserId: euaUserId.toUpperCase(),
-          component,
-          role,
-          systemIntakeId
+    return (
+      createSystemIntakeContact({
+        variables: {
+          input: {
+            euaUserId: euaUserId.toUpperCase(),
+            component,
+            role,
+            systemIntakeId
+          }
         }
-      }
-    })
-      .then(refetch)
-      .then((response: ApolloQueryResult<GetSystemIntakeContacts>) =>
-        response?.data?.systemIntakeContacts?.systemIntakeContacts.find(
-          obj => obj.role === role
+      })
+        // Refetch contacts
+        .then(refetch)
+        // Return new contact
+        .then((response: ApolloQueryResult<GetSystemIntakeContacts>) =>
+          response?.data?.systemIntakeContacts?.systemIntakeContacts.find(
+            obj => obj.role === role
+          )
         )
-      );
+        // If error, return initial contact
+        .catch(() => contact as AugmentedSystemIntakeContact)
+    );
   };
 
-  /** Update system intake contact in database */
+  /**
+   * Update system intake contact in database
+   * */
   const updateContact = async (contact: SystemIntakeContactProps) => {
     const { id, euaUserId, component, role } = contact;
-    return updateSystemIntakeContact({
-      variables: {
-        input: {
-          id,
-          euaUserId: euaUserId.toUpperCase(),
-          component,
-          role,
-          systemIntakeId
+    return (
+      updateSystemIntakeContact({
+        variables: {
+          input: {
+            id,
+            euaUserId: euaUserId.toUpperCase(),
+            component,
+            role,
+            systemIntakeId
+          }
         }
-      }
-    })
-      .then(refetch)
-      .then(
-        (response: ApolloQueryResult<GetSystemIntakeContacts>) =>
-          response?.data?.systemIntakeContacts?.systemIntakeContacts
-      );
+      })
+        // Refetch contacts
+        .then(refetch)
+        // Return updated contacts
+        .then(
+          (response: ApolloQueryResult<GetSystemIntakeContacts>) =>
+            response?.data?.systemIntakeContacts?.systemIntakeContacts
+        )
+        // If error, return existing contacts
+        .catch(() => data?.systemIntakeContacts?.systemIntakeContacts)
```

Comment:
I think this could be shortened to be `systemIntakeContacts` as it is defined above